### PR TITLE
Remove job from container stats

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

**Comment by LK4D4** [comment]
@runcom Can you split your changes to two commits, pls?

**Comment by runcom** [comment]
Sure!

File: integration-cli/docker_api_resize_test.go [hunk 1]
```diff
@@ -24,6 +24,24 @@ func TestResizeApiResponse(t *testing.T) {
...
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		t.Fatalf(out, err)
+	}
+	defer deleteAllContainers()
+	cleanedContainerID := strings.TrimSpace(out)
+
+	endpoint := "/containers/" + cleanedContainerID + "/resize?h=foo&w=bar"
+	_, err = sockRequest("POST", endpoint, nil)
+	if err == nil {
```
Comment (HuKeping) [inline integration-cli/docker_api_resize_test.go]: maybe `if err != nil`?

Comment (runcom) [inline integration-cli/docker_api_resize_test.go]: no, if err is nil it means api call was ok, i'm looking for failure here

Comment (HuKeping) [inline integration-cli/docker_api_resize_test.go]: Oops~, i misunderstand that, ignore my noise

File: api/server/server.go [hunk 2]
```diff
@@ -543,10 +543,10 @@ func getContainersStats(eng *engine.Engine, version version.Version, w http.Resp
...
 		return fmt.Errorf("Missing parameter")
 	}
-	name := vars["name"]
-	job := eng.Job("container_stats", name)
-	streamJSON(job, w, true)
-	return job.Run()
+
+	d := getDaemon(eng)
+
+	return d.ContainerStats(vars["name"], utils.NewWriteFlusher(w))
```
Comment (cpuguy83) [inline api/server/server.go]: Creating a flusher here but `daemon.ContainerStats` only accepts an `io.Writer`?

Comment (runcom) [inline api/server/server.go]: Missed that

Comment (LK4D4) [inline api/server/server.go]: it should be `io.Writer`. `Flush` called automatically on each write.

**Comment by LK4D4** [comment]
LGTM

**Comment by cpuguy83** [comment]
LGTM

**Pull request merged**